### PR TITLE
Add Arbitrary[Instant].

### PR DESCRIPTION
**Pull request by kevinmeredith**
Adds an `Arbitrary[Instant]`. I didn't see one in this library, so I opened this PR!

### CODE REVIEW

**Comment by kevinmeredith** [comment]
The failed builds show:

```
The command "~/bin/install-jdk.sh --target "/home/travis/oraclejdk8" --workspace "/home/travis/.cache/install-jdk" --feature "8" --license "BCL"" failed and exited with 3 during .
```

My guess is that there's an issue with the build?

**Comment by fedefernandez** [comment]
@kevinmeredith thanks, I think this could solve that
https://github.com/47deg/scalacheck-toolbox/pull/65

**Comment by fedefernandez** [comment]
@kevinmeredith could you please update your branch with master? Thanks

**Comment by kevinmeredith** [comment]
> @kevinmeredith could you please update your branch with master? Thanks

Hey Fede - sure, thanks!

File: datetime/src/main/scala/com/fortysevendeg/scalacheck/datetime/jdk8/GenJdk8.scala [hunk 1]
```diff
@@ -54,4 +53,14 @@ object GenJdk8 extends GenJdk8
...
+
+  implicit val arbInstant: Arbitrary[Instant] = {
+
+    implicit val c: Choose[Instant] =
+      Choose.xmap[Long, Instant](Instant.ofEpochSecond(_), _.getEpochSecond)
+
+    Arbitrary(
+      Gen.choose[Instant](Instant.MIN, Instant.MAX)
+    )
+  }
```
Comment (fedefernandez) [inline datetime/src/main/scala/com/fortysevendeg/scalacheck/datetime/jdk8/GenJdk8.scala]: In order to use the granularity we could write something like:

```scala
implicit def arbInstant(implicit zdtArb: Arbitrary[ZonedDateTime]): Arbitrary[Instant] =
  Arbitrary(zdtArb.arbitrary.map(_.toInstant))
```

Comment (kevinmeredith) [reply datetime/src/main/scala/com/fortysevendeg/scalacheck/datetime/jdk8/GenJdk8.scala]: Is there an `Arbitrary[ZonedDateTime]` already in scope, @fedefernandez, such that we don't need the `implicit zdtArb: Arbitrary[ZonedDateTime]`?

I could be missing something too in your suggestion.

Comment (fedefernandez) [reply datetime/src/main/scala/com/fortysevendeg/scalacheck/datetime/jdk8/GenJdk8.scala]: hey @kevinmeredith I just found an old branch with this instance and some others (`LocalDateTime` and `LocalDate`). I've cleaned up a bit and create a PR, feel free to take a look

https://github.com/47deg/scalacheck-toolbox/pull/66

Comment (kevinmeredith) [reply datetime/src/main/scala/com/fortysevendeg/scalacheck/datetime/jdk8/GenJdk8.scala]: 👍

**Comment by kevinmeredith** [comment]
Closing to https://github.com/47deg/scalacheck-toolbox/pull/66

**Pull request closed without merging**